src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -1727,7 +1727,7 @@ class device {
             if(map->m_mem_data[i].m_type == MEM_STREAMING)
                 continue;
 
-            if(!strncmp((const char*)map->m_mem_data[i].m_tag, "HOST", 4))
+            if(isHostMem((const char*)map->m_mem_data[i].m_tag))
```

Comment:
No C-style casts please.  Not something you changed :-) but maybe you can fix it?